lib/adapters/FileSystem.js

```diff
@@ -28,11 +29,14 @@ class FileSystem extends AbstractAdapter {
 	 * @param {string} parameters.virBasePath Virtual base path
 	 * @param {string} parameters.fsBasePath (Physical) File system path
 	 * @param {string[]} [parameters.excludes] List of glob patterns to exclude
-	 * @param {object} [parameters.project] Experimental, internal parameter. Do not use
+	 * @param {object} [parameters.useGitignore=false]
+	 *   Whether to apply any excludes defined in an optional .gitignore in the base directory
```

Comment:
should we be here more precise by naming the base directory
```suggestion
	 *   Whether to apply any excludes defined in an optional .gitignore in the given <code>fsBasePath</code> directory
```